### ISSUE TITLE
reusable components

### ISSUE THREAD

**Issue by EricWVGG** [diegohaz/arc#280]
I have a project that requires reusable components… they look something like this:

`<Article id="1" /><Article id="2" />`

Obviously the reducer logic updates both components with the latest API call. 

I have found a promising guide to encapsulating logic here — https://www.codementor.io/nwugurusunday/implement-a-reusable-reducer-logic-5eqjahjum — but the store is handled completely differently than ARC. 

As a beginner, I'm a bit stumped. Am I overlooking something built-in? Is there a more correct way to accomplish this?

**Comment by Geczy** [comment]
Not an issue for arc...

Call the fetch once, and loop the list. Same as PostList is doing it here in arc

**Comment by EricWVGG** [comment]
The articles are not in a series like a blog. Its a bit more ad hoc like a magazine. 

`<Article id="1"/><SomeOtherContentType id="1"/><SomethingElse id="123"/><Article id="2"/>

Something like that

/taptapswipe

**Comment by EricWVGG** [comment]
I've crashed into this problem a second time. I have the equivalent of two PostLists in a single page.

**Comment by EricWVGG** [comment]
Has anyone tried using redux-subspace with ARC?

https://github.com/ioof-holdings/redux-subspace#reducers

**Comment by EricWVGG** [comment]
Nope. I’m just got a dozen different stores for various purposes. It’s a mess.
